gen2-bounding-box-filtering/kalman/errors.csv

```diff
@@ -0,0 +1,20 @@
+﻿Ground truth,OAK-D Error [%]

```

Comment:
Maybe we could just use theoretical depth error for this estimation ([here](https://docs.google.com/spreadsheets/d/1ymn-0D4HcCbzYP-iPycj_PIdSwmrLenlGryuZDyA4rQ/edit#gid=0))